webpack.config.js

```diff
@@ -218,8 +218,8 @@ const webworkerConfig = {
     new CopyPlugin({
       patterns: [
         {
-          from: path.resolve(browserPath, 'fragments', 'ably.d.ts'),
-          to: path.resolve(browserPath, 'static', 'ably-webworker.min.d.ts'),
+          from: path.resolve(browserPath,  'fragments', 'ably.d.ts'),
+          to: path.resolve(browserPath, 'build', 'ably-webworker.min.d.ts'),
```

Comment:
I would guess that `build` is top level directory, rather than a child of browserPath

Reply:
The webpack config was mostly entirely rewritten in [another commit](https://github.com/ably/ably-js/pull/982/files#diff-1fb26bc12ac780c7ad7325730ed09fc4c2c3d757c276c3dacc44bfe20faf166f)